massa-bootstrap/src/lib.rs

```diff
@@ -29,6 +32,25 @@ mod messages;
 mod server_binder;
 pub mod settings;
 
+/// a summary of the bootstrap state snapshots
+#[derive(Default, Debug)]
+pub struct BootstrapStateSummary {
```

Comment:
It's not a summary, it's the full bootstrap state

Reply:
renamed it